python-lib/dku_config/dss_parameter.py

```diff
@@ -91,7 +91,7 @@ def print_success_message(self):
         logger.info('All checks have been successfully done for {}.'.format(self.name))
 
     def __repr__(self):
-        return "DSSParameter(name={}, value={})".format(self.name, self.value)
+        return str({"value": self.value})
 
     def __str__(self):
-        return "DSSParameter(name={}, value={})".format(self.name, self.value)
+        return str({"value": self.value})
```

Comment:
Mmmh, I did that for debugging purposes. If I want to debug the plugin in a jupyter-notebook, I need to build my recipe with a valid dku_config. To do so, I print the `dku_config` when executing the recipe in dss, and I just have to copy/paste what is printed in the logs to use it in my Jupyter Notebook.
In other words, when doing a `print(self.dku_config)`, I want the printed text to be copiable, so that I just have to do `dku_config = DkuConfig(**copied_dku_config_from_log)` in my jupyter-notebook and it works.
Do you think I should rather add a function `print_as_dict()` for debugging?